modules/shared/components/organisms/ImagePollView/ImagePollView.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import PostViewHeader from '../../molecules/PostViewHeader/PostViewHeader';
+import PostViewFooter from '../../molecules/postFooter/PostFooter';
+import type { IImagePollView } from './IImagePollView';
+import ImagePollGroup from '../../molecules/ImagePollGroup/ImagePollGroup';
+
+const ImagePollView: FC<IImagePollView.IProps> = ({ post }): ReactElement => {
+  const firstIndex = 0;
+  return (
+    <div className="bg-white p-m shadow-soft rounded-md space-y-4" id={post.id}>
+      <PostViewHeader
+        id={post.id}
+        date={new Date(post.created_at)}
```

Comment:
we need to be more explicit about the date `naming`, which date it refers to?
- matchDate, birthDate,....etc